buildSrc/src/main/java/org/opensearch/gradle/PublishPlugin.java

```diff
@@ -120,23 +120,25 @@ public String call() throws Exception {
                 });
             });
         });
-
         publishing.getPublications().withType(MavenPublication.class, publication -> {
-            // Add git origin info to generated POM files
-            publication.getPom().withXml(PublishPlugin::addScmInfo);
-
-            // have to defer this until archivesBaseName is set
-            project.afterEvaluate(p -> publication.setArtifactId(getArchivesBaseName(project)));
-
-            // publish sources and javadoc for Java projects.
-            if (project.getPluginManager().hasPlugin("opensearch.java")) {
-                publication.artifact(project.getTasks().getByName("sourcesJar"));
-                publication.artifact(project.getTasks().getByName("javadocJar"));
+            String search  = "zip";
```

Comment:
Hi @prudhvigodithi,

A bit confused.

Can we have a better naming for this var?
`search` does not tell me anything on it.

And the logic seems to find whether these is any zips, before attempting the publish of java later.

Thanks.

Reply:
Hi @peterzhuamazon 
Fixed access modifiers and var strings in my latest commit.